### ISSUE TITLE
Pytorch implementation or Keras implementation 

### ISSUE THREAD

**Issue by carrymybaby** [ZJU-DAI/Federated-Mutual-Learning#3]
Hello, I recently saw your paper about FML, in which the interaction of loss function is very enlightening. In the code, you completed the model based on pytorch. Is there any implementation thoughts based on Keras? Because it seems impossible to implement manual backward in keras
